src/coreComponents/mesh/Perforation.cpp

```diff
@@ -51,6 +52,19 @@ Perforation::Perforation( string const & name, Group * const parent )
     setRTTypeName( rtTypes::CustomTypes::groupNameRef ).
     setInputFlag( InputFlags::OPTIONAL ).
     setDescription( "Target region to connect the perforation" );
+
+  registerWrapper( viewKeyStruct::perfStatusTableNameString(), &m_perfStatusTableName ).
+    setRTTypeName( rtTypes::CustomTypes::groupNameRef ).
+    setInputFlag( InputFlags::OPTIONAL ).
+    setDescription(
+    "Name of table function defining perforation status as a function of time. If none specified, a table function with be created internally and assigned the same name as the perforation." );
```

Comment:
Maybe add precisions / example:
```suggestion
    "Name of table function defining perforation status as a function of time:\n"
    "- The first row sets the perforation time coordinates,\n"
    "- The following row is a multiplier of the perforation rate (typically, 0.0 to close, 1.0 to open) for each time coordinate.\n"
    "If none specified, a table function with be created internally and assigned the same name as the perforation.\n"
    "A table that opens the perforation at t=100, and closes it at t=200 could be `{{ 0, 100, 200 },{ 0.0, 1.0, 0.0 }}`" );
```

Reply:
Thanks ... good suggestion I've added the description with a slight modification. The intent of perf status is to be an on/off indicator, with potentially other statuses as needed for well actions.   The fractional value I think you are referring to would be a PI multiplier field.